### PR TITLE
Remove leftover experimental flags

### PR DESCRIPTION
**Pull request by found-it**
The experimental flag for images is a leftover relic from an older time. We have focused efforts around testing and image quality now so I'm dropping the leftover experimental flags as we haven't been tracking them for a while now.

All of these images have some level of basic testing, we are tracking test categoric test improvements elsewhere. I think it is safe to remove the experimental flag here and continue with our regular efforts for adding tests.

### CODE REVIEW

File: images/rust/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <!--monopod:end-->
 
 <!--overview:start-->
-Minimal Wolfi-based Rust image for building Rust applications. **EXPERIMENTAL**.
+Minimal Wolfi-based Rust image for building Rust applications..
```
Comment (stormqueen1990) [inline images/rust/README.md:16]: nit: extra dot
```suggestion
Minimal Wolfi-based Rust image for building Rust applications.
```

File: images/rust/metadata.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ image: cgr.dev/chainguard/rust
 logo: https://storage.googleapis.com/chainguard-academy/logos/rust.svg
 endoflife: ""
 console_summary: ""
-short_description: Minimal Wolfi-based Rust image for building Rust applications. **EXPERIMENTAL**.
+short_description: Minimal Wolfi-based Rust image for building Rust applications..
```
Comment (stormqueen1990) [inline images/rust/metadata.yaml:6]: nit: extra dot
```suggestion
short_description: Minimal Wolfi-based Rust image for building Rust applications.
```

**Review by stormqueen1990 — DISMISSED** [review]
LGTM, just a few grammar nits

**Comment by joshrwolf** [comment]
the failures look like real failures that are on HEAD that are unrelated to this PR, so I'm going to make the case for force merging this

**Review by joshrwolf — APPROVED** [review]

**Review by stormqueen1990 — DISMISSED** [review]

**Review by imjasonh — APPROVED** [review]

**Comment by imjasonh** [comment]
Force merging past unrelated test failures.

**Pull request merged**